src/common/enum_util.cpp

```diff
@@ -441,6 +441,7 @@ ArrowOffsetSize EnumUtil::FromString<ArrowOffsetSize>(const char *value) {
 
 const StringUtil::EnumStringLiteral *GetArrowTypeInfoTypeValues() {
 	static constexpr StringUtil::EnumStringLiteral values[] {
+		{ static_cast<uint32_t>(ArrowTypeInfoType::NONE), "NONE" },
```

Comment:
This change is not necessary, we don't use the `NONE` variant of the ArrowTypeInfoType anywhere